test/unit/steps/compliance/dwp-issuing-office/DWPIssuingOffice.test.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const { expect } = require('test/util/chai');
+const DateUtils = require('utils/DateUtils');
+const DWPIssuingOffice = require('steps/compliance/dwp-issuing-office/DWPIssuingOffice');
+const paths = require('paths');
+
+describe('DWPIssuingOffice.js', () => {
+
+    let dWPIssuingOffice;
+
+    beforeEach(() => {
+
+        dWPIssuingOffice = new DWPIssuingOffice({
+            journey: {
+                steps: {
+                    MRNDate: paths.compliance.mrnDate
+                }
+            }
+        });
+    });
+
+    describe('get path()', () => {
+
+        it('returns path /dwp-issuing-office', () => {
+            expect(dWPIssuingOffice.path).to.equal(paths.compliance.dwpIssuingOffice);
+        });
+
+    });
+
+    describe('get form()', () => {
+
+        let fields;
+        let field;
+
+        beforeEach(() => {
+            fields = dWPIssuingOffice.form.fields;
+        });
+
+        after(() => {
```

Comment:
There's no need to define `after` in each of our tests, moreover, I've removed all of these from our tests. I'm unsure if the GC will run during a test cycle, even if it did the amount of memory freed would be negligible considering the number of tests we have.

Reply:
@paulgain after still exists in all the tests for the fields. If you want, I can remove them all

Reply:
Yes please, I have removed quite a few. Feel free to clear up. Thank you.